versions.yaml

```diff
@@ -135,10 +135,11 @@ assets:
           version: *default-image-version
       x86_64:
         name: *default-image-name
-        version: *default-image-version
+        version: "oracular"
```

Comment:
I was wondering why an explicit assignment is made here even if `default-image-version` is set to `oracular`.

Reply:
This has been answered above. Thanks.